test/specs/commands/createTransactionCastVotes.js

```diff
@@ -0,0 +1,259 @@
+/*
+ * LiskHQ/lisky
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { setUpCommandcreateTransactionCastVotes } from '../../steps/setup';
+import * as given from '../../steps/1_given';
+import * as when from '../../steps/2_when';
+import * as then from '../../steps/3_then';
+
+describe('create transaction cast votes', () => {
+	beforeEach(setUpCommandcreateTransactionCastVotes);
+	Given('a Vorpal instance with a UI and an active command that can prompt', given.aVorpalInstanceWithAUIAndAnActiveCommandThatCanPrompt, () => {
+		Given('an action "create transaction cast votes"', given.anAction, () => {
+			Given('a passphrase "minute omit local rare sword knee banner pair rib museum shadow juice"', given.aPassphrase, () => {
+				Given('a second passphrase "fame spoil quiz garbage mirror envelope island rapid lend year bike adapt"', given.aSecondPassphrase, () => {
+					Given('a Lisk object that can create transactions', given.aLiskObjectThatCanCreateTransactions, () => {
+						Given('the passphrase can be retrieved from its source', given.thePassphraseCanBeRetrievedFromItsSource, () => {
```

Comment:
I think this block is redundant. Right?

Reply:
Ah sorry, this is testing a single public key rather than more than one. 👍

Reply:
Maybe we need a distinct step to make this clear: `a public key that should be voted for`.

Reply:
Hm, this would add much boilerplate also in other functions. As you can add many or just one public key to the `votes` option this makes sense to me so far.